spec/unit/lib/cloud_controller/rack_app_builder_spec.rb

```diff
@@ -216,6 +216,28 @@ module VCAP::CloudController
           expect(CloudFoundry::Middleware::CefLogs).to have_received(:new).with(anything, fake_logger, TestConfig.config_instance.get(:local_route))
         end
       end
+
+      describe 'Deprecated CF CLI Warning' do
+        before do
+          allow(CloudFoundry::Middleware::BelowMinCliWarning).to receive(:new)
+        end
+
+        context 'with min_cf_cli_version provided' do
+          before do
+            builder.build(TestConfig.override(info: { min_cli_version: '7.0.0' }, warn_if_below_min_cli_version: true), request_metrics, request_logs).to_app
+          end
+
+          it 'enables the DeprecatedCfCliWarning middleware' do
```

Comment:
I guess you renamed the file from `DeprecatedCfCliWarning` to `BelowMinCliWarning`? Should be adapted.